<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>0</y>
      <width>631</width>
      <height>723</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Debt Simplifier</string>
     </attribute>
     <widget class="QSplitter" name="splitter_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>301</width>
        <height>291</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Number of people</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nameLineEdit"/>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </widget>
      <widget class="QTableWidget" name="tableWidget"/>
      <widget class="QSplitter" name="splitter_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="pushButton_20">
        <property name="text">
         <string>Load Previous Config</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_21">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_9">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>256</width>
        <height>408</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Original balances</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget"/>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Reduced balances</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_2"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Travel Helper</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of locations</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>256</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_4">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>50</y>
        <width>256</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Shortest distance between each location</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculate Minimum Distance</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>260</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculate Minimum Path</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>230</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>290</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>Best Combination</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>205</width>
        <height>50</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Item Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Price</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_4"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_12">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_2">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>11</y>
        <width>202</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Max Budget</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2"/>
      <widget class="QPushButton" name="pushButton_11">
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>241</width>
        <height>204</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Items</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_5"/>
     </widget>
     <widget class="QSplitter" name="splitter_4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>170</y>
        <width>256</width>
        <height>204</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Combinations</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_6"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_10">
     <attribute name="title">
      <string>Expense Tracker</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_7">
      <property name="geometry">
       <rect>
        <x>254</x>
        <y>10</y>
        <width>254</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_18">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Previous Config</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_19">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>311</width>
        <height>135</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Maintanence</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Grocieries</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QProgressBar" name="progressBar_2">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Bills</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QProgressBar" name="progressBar_3">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Miscellaneous</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QProgressBar" name="progressBar_4">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>292</width>
        <height>100</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_5"/>
       </item>
       <item row="0" column="2" rowspan="3">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_14">
           <property name="text">
            <string>Add Transaction</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_15">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Category</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Amount</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_7"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_2"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>632</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
